gnucash/python/init.py

```diff
@@ -99,7 +99,7 @@ def refresh (self):
     def quit (self):
         """ quit """
 
-        self.write("\nHave a nice day !\n")
+        self.write(_("\nHave a nice day !\n"))
```

Comment:
AFAiK only in French spaces before punctuation marks are in some cases common, in English not.

Reply:
... and in this specific I would prefer to reuse
28877:msgid "Have a nice day!"
and move the line feeds out of the translatable string.